sdk/python/feast/registry.py

```diff
@@ -143,44 +213,60 @@ def clone(self) -> "Registry":
         new_registry._registry_store = NoopRegistryStore()
         return new_registry
 
+    def to_repo_contents(self, project: str) -> RepoContents:
+        """
+        Convert the contents of the registry for the given project into a RepoContents object.
+
+        Args:
+            project: The Feast project to be converted.
+        """
+        return RepoContents(
+            entities=set(self.list_entities(project=project)),
+            feature_views=set(self.list_feature_views(project=project)),
+            request_feature_views=set(self.list_request_feature_views(project=project)),
+            on_demand_feature_views=set(
+                self.list_on_demand_feature_views(project=project)
+            ),
+            feature_services=set(self.list_feature_services(project=project)),
+        )
+
     # TODO(achals): This method needs to be filled out and used in the feast plan/apply methods.
     @staticmethod
     def diff_between(
```

Comment:
this method seems to logically belong as a method within RegistryDiff or something. You no longer are passing in registry protos at least

Reply:
yup this is fair - I moved this method to `FcoDiff.py`